src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -1427,6 +1444,7 @@ public void loadProblemInformationAndDataFiles(IInternalContest contest, String
             pc2FormatProblemYamlFile = true;
         }
 
+        // TODO: I am not sure why this test is contingent on pc2FormatProblemYamlFile. (JB)
```

Comment:
I do understand what the 'if' _does_, but I think it's silly.  If the title is `null`, then just set it to the `name `in the **yaml** file, right?  Why does it have to be a **pc2FormatProblemYamlFile** IMHO it has nothing to do with being a **pc2FormatProblemYamlFile**?  If '`name`' is in the file, use it.  The very next statement would cause it to abort out anyway if it's **null**, why not avoid that?